src/main/java/tr/ogu/edu/school/schoolapp/security/SecurityConfig.java

```diff
@@ -48,4 +53,16 @@ public AuthenticationManager authenticationManager(AuthenticationConfiguration a
 	public PasswordEncoder passwordEncoder() {
 		return new BCryptPasswordEncoder();
 	}
+
+	@Bean
+	public CorsConfigurationSource corsConfigurationSource() {
+		CorsConfiguration configuration = new CorsConfiguration();
+		configuration.setAllowedOrigins(Arrays.asList("http://localhost:19006"));
```

Comment:
19006 port herkes için geçerli mi, bende 3000 olabilir mi?

Reply:
hayır hocam default 19006 olarak açılıyor. 
configuration.setAllowedOrigins(Arrays.asList("*"));
bu şekilde yazarak tüm adresleri kabul etmek istememiştim.